### PR TITLE
vimtutor-sequel 1.3.1 (new formula)

### PR DESCRIPTION
**Pull request by micahkepe**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/v/vimtutor-sequel.rb
Comment (SMillerDev) [inline Formula/v/vimtutor-sequel.rb:18]: This is not specific to Homebrew so it should not be a caveat.

**Comment by micahkepe** [comment]
Hi,

I have removed the caveats section as per the feedback and fixed some formatting issues in the formula. After these changes, the formula is now passing all the audits and required checks.

However, the "CI / Linux (pull_request)" check is still failing, but this check does not appear to be required for merging. Could you please advise whether this is something that needs to be addressed before the formula can be merged, or if the PR can proceed as is?

Thank you for your guidance!

**Comment by SMillerDev** [comment]
> is now passing all the audits and required checks.

It's actually failing on Linux on one of those. The other platforms are simply not run because one already failed.

**Comment by micahkepe** [comment]
Hello,
     I have been trying to address the failed Linux test but am still not sure what is causing the failed test. I did the following to try to troubleshoot and fix the issue:

1. Added 
```
depends_on "vim"
```

2. I tried tapping the test-bot as well and ran the following:
```
brew tap homebrew/test-bot
brew test-bot --only-cleanup-before
brew test-bot --only-formulae --only-json-tab --skip-recursive-dependents vimtutor-sequel
```
And these steps passed.

**Comment by SMillerDev** [comment]
The error isn't something you can fix with changes to the formula.
```
Full audit --formula vimtutor-sequel --online --new output
  vimtutor-sequel
    * GitHub repository too new (<30 days old)
```

File: Formula/v/vimtutor-sequel.rb
Comment (chenrui333) [inline Formula/v/vimtutor-sequel.rb:18]: seems a bit redundant checks

```suggestion
```

Comment (micahkepe) [reply Formula/v/vimtutor-sequel.rb:18]: Would just having the following be sufficient for the test block?
```
test do
    assert_match "Vimtutor Sequel version #{version}", shell_output("#{bin}/vimtutor-sequel --version")
end
```

Comment (micahkepe) [reply Formula/v/vimtutor-sequel.rb:18]: Removed the redundant code

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@micahkepe, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**
